src/pages/login.tsx

```diff
@@ -1,60 +1,173 @@
-import React from "react";
-import { TextField, Button, CircularProgress } from "@mui/material";
+import { TextField, Button, Grid, Box, Paper, Typography, Link, Container } from "@mui/material";
 import { styled } from "@mui/material/styles";
-import { useInput } from "@/hook/forms";
-import toast from "react-hot-toast";
+import { FcGoogle } from "react-icons/fc";
+import { useInput } from "@/util/forms";
+// import { Toast } from "@/util/notifications";
 import { Auth } from "aws-amplify";
-import NextLink from "next/link";
-import { useRouter } from "next/router";
 
-const Field = styled(TextField)({
-  margin: "10px 0",
+const StyledTextField = styled(TextField)({
+  borderRadius: "5px",
+  background: "transparent",
 });
 
-const Link = styled(NextLink)({
-  margin: "15px 0",
-  textAlign: "right",
-});
-
-const Signup: React.FC = () => {
-  const router = useRouter();
-  const [loading, setLoading] = React.useState(false);
-
+const SignIn: React.FC = () => {
   const { value: email, bind: bindEmail } = useInput("");
   const { value: password, bind: bindPassword } = useInput("");
 
   const handleSubmit = async (e: React.SyntheticEvent<Element, Event>) => {
     e.preventDefault();
-    setLoading(true);
-
     try {
-      const resp = await Auth.signIn(email, password);
-      console.log(resp);
-    } catch (error) {
-      toast.error("error toasted!");
-    }
-    setLoading(false);
+      const res = await Auth.signIn(email, password);
+      console.log(res);
+    } catch (error) {}
   };
 
   return (
-    <form
-      style={{
-        display: "flex",
-        flexDirection: "column",
-        justifyContent: "space-between",
-      }}
-      onSubmit={handleSubmit}
-    >
-      <h1 style={{ fontSize: "22px", fontWeight: 800 }}> Sign in to an existing account</h1>
-      <Field label="Email" {...bindEmail} type="email" />
-      <Field label="Password" type="password" {...bindPassword} />
-      <Button variant="contained" color="primary" size="large" type="submit" disabled={loading}>
-        {loading && <CircularProgress size={20} style={{ marginRight: 20 }} />}
-        Login to Your Account
-      </Button>
-      <Link href={`${router.basePath}/signup`}>Make a new account &rarr;</Link>
-    </form>
+    <Container>
+      <Grid container component="main" sx={{ height: "100vh", flexWrap: "wrap" }} spacing={2}>
+        <Grid
+          item
+          xs={12}
+          sm={8}
+          md={6}
+          component={Paper}
+          elevation={6}
+          square
+          alignItems="center"
+          display="flex"
+        >
+          <Box
+            sx={{
+              my: 8,
+              mx: 4,
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "flex-start",
+            }}
+          >
+            <Box
+              sx={{
+                mb: 1,
+              }}
+            >
+              <img
+                src={`/png/favicon.png`}
```

Comment:
change this back to favicon-32x32 
![image](https://user-images.githubusercontent.com/68600416/194757492-1fa97f4f-2b35-427c-a151-74c3602f0f0e.png)
Don't rename this